release/package.json

```diff
@@ -758,6 +758,20 @@
           "description": "Enables detection of cases when names of functions and values can be simplified",
           "type": "boolean"
         },
+        "FSharp.simplifyNameAnalyzerExclusions": {
+          "default": [
+            ".*\\.g\\.fs",
+            ".*\\.cg\\.fs"
```

Comment:
Since the issue was based on code generated files, this is the conventions I've seen in the wild for good defaults.

Reply:
thought for this - we should probably exclude anything in the intermediate output directory for the project. How are these regexes uses? Are they evaluated per-project?

Reply:
The analyzers are run after each typecheck of a file and this is added to the conditional whether the analyzer is active or doesn’t match one of these reflexes.

Reply:
So if we wanted to do obj/*.fs we could add it easily.

Reply:
I think intermediate dir would have to be more 'dynamic' or FSAC-included, because it's an MSBuild property that can and does change. .NET 8 adds a new optional directory layout that drastically changes it, for example.

Reply:
Other option is to just include things we _know_ shouldn't be analyzed like AssemblyInfo.fs.  Otherwise I think this may need more changes to FSAC to get a more dynamic Intermediate folder knowledge.